From foo@baz Wed Aug 19 16:20:41 PDT 2009
Date: Wed, 19 Aug 2009 16:20:41 -0700
To: Greg KH <greg@kroah.com>
From: Greg Kroah-Hartman <gregkh@suse.de>
Subject: Staging: hv: remove include/HvHalApi.h

It isn't needed at all, was only being used for one typedef,
which is now removed.

Cc: Hank Janssen <hjanssen@microsoft.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/staging/hv/Hv.h                 |    1 -
 drivers/staging/hv/include/HvHalApi.h   |   32 --------------------------------
 drivers/staging/hv/include/HvSynicApi.h |    4 ++--
 3 files changed, 2 insertions(+), 35 deletions(-)

--- a/drivers/staging/hv/Hv.h
+++ b/drivers/staging/hv/Hv.h
@@ -26,7 +26,6 @@
 #define __HV_H__
 
 #include "include/HvStatus.h"
-#include "include/HvHalApi.h"
 #include "include/HvVpApi.h"
 #include "include/HvSynicApi.h"
 #include "include/HvHcApi.h"
--- a/drivers/staging/hv/include/HvHalApi.h
+++ /dev/null
@@ -1,32 +0,0 @@
-/*
- *
- * Copyright (c) 2009, Microsoft Corporation.
- *
- * This program is free software; you can redistribute it and/or modify it
- * under the terms and conditions of the GNU General Public License,
- * version 2, as published by the Free Software Foundation.
- *
- * This program is distributed in the hope it will be useful, but WITHOUT
- * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
- * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
- * more details.
- *
- * You should have received a copy of the GNU General Public License along with
- * this program; if not, write to the Free Software Foundation, Inc., 59 Temple
- * Place - Suite 330, Boston, MA 02111-1307 USA.
- *
- * Authors:
- *   Haiyang Zhang <haiyangz@microsoft.com>
- *   Hank Janssen  <hjanssen@microsoft.com>
- *
- */
-
-
-#pragma once
-
-
-
-/* Time in the hypervisor is measured in 100 nanosecond units */
-
-typedef u64 HV_NANO100_TIME,     *PHV_NANO100_TIME;
-typedef u64 HV_NANO100_DURATION, *PHV_NANO100_DURATION;
--- a/drivers/staging/hv/include/HvSynicApi.h
+++ b/drivers/staging/hv/include/HvSynicApi.h
@@ -298,8 +298,8 @@ typedef struct _HV_TIMER_MESSAGE_PAYLOAD
 {
     u32          TimerIndex;
     u32          Reserved;
-    HV_NANO100_TIME ExpirationTime;     /* When the timer expired */
-    HV_NANO100_TIME DeliveryTime;       /* When the message was delivered */
+    u64 ExpirationTime;     /* When the timer expired */
+    u64 DeliveryTime;       /* When the message was delivered */
 } HV_TIMER_MESSAGE_PAYLOAD, *PHV_TIMER_MESSAGE_PAYLOAD;
 
 
